etc/dosemu.conf

```diff
@@ -765,6 +765,16 @@
 
 # $_SDL_hwrend = (off)
 
+# Name of the TTF font that is used for SDL2 (e.g. "vga")
+# Default = "Flexi IBM VGA9 True"
+
+# $_SDL_font = "Flexi IBM VGA9 True"
+
+# Size of the TTF font in points
+# Default: 22
+
+# $_SDL_font_size = (22)
```

Comment:
Do we really need this?
If you have the code that can scale dynamically
(dynamically reload the font with needed size),
why do we need to specify the initial value?
I think it should just be taken for the default window
size, no?

Reply:
I think the default window size is based on text mode geometry x charsize. In bdf both the style and size are defined by the choice of font, however with ttf being scalable we need an initial size too, and it has a good analogue to how a usual Linux terminal window's size is determined.

Reply:
> I think the default window size is based on text mode geometry x charsize.

In an X_font mode yes, but that's a discrepancy.
As you made the resizable font, we can have the
initial size based on vga.char_width, vga.char_height
again. And for altering the initial size you already
have `$_X_winsize`, so why to add another contradicting
option.

Reply:
I removed this and resized to best fit to X_winsize.